app/interactors/add_movie_to_round.rb

```diff
@@ -0,0 +1,28 @@
+require 'hanami/interactor'
+
+class AddMovieToRound
+  include Hanami::Interactor
+
+  attr_reader :round, :tmdb_id
```

Comment:
To be compliant with the official "way of the Interactor", internal parameters and computations should be accessed via the instance variables (`@...`) and we should `expose` the name of those variables that can be accessed in the `Result`.

Reply:
We don't really want to expose any variables, so I just removed the `attr_reader`.